### ISSUE TITLE
CLI option to display/hide citation?

### ISSUE THREAD

**Issue by Caulm** [bsdayo/BingChat#24]
I noticed that the CLI reply containes citation numbers, like "[^1^]", but does not contain the source attribution link. It looks a bit strange. I'm not sure if this is intentional.

Does it need an option to control whether citations are displayed or hidden?

- displayed: For users who want to read the source page for detailed information.
- hidden: For users who want to chat or get direct answers.

**Comment by neon-sunset** [comment]
Yeah, this is https://github.com/bsdayo/BingChat/blob/main/src/BingChat/BingChatConversation.cs#L98-L99
